deploy/examples/monitoring/prometheus.yaml

```diff
@@ -21,19 +21,19 @@ metadata:
   labels:
     rbac.ceph.rook.io/aggregate-to-prometheus: "true"
 rules:
-- apiGroups: [""]
-  resources:
-  - nodes
+  - apiGroups: [""]
+    resources:
+      - nodes
   - services
```

Comment:
This still doesn't look correct. Previously, `- services` was at the same indentation as `- nodes`